### DISCUSSION TITLE
Velocity scaling

### DISCUSSION THREAD

**Discussion by Blitzboot** [kroshu/kuka_drivers#189] [Q&A] [answered]
Hello, 
i have one more question, is there an option for velocity scaling in the controller? 
I know that there ist only an joint angle control for kuka kss Driver. 
But is it the best way to just publish the points in the velocity i like or is there a more elegant way?

Thanks for your answers.

**Comment by Svastits** [comment, marked as answer]
Hi, there is no velocity scaling, the joint trajectory controller executes the trajectory it receives.
If you use moveit, you can easily scale the velocity using the move group interface.
We also do not plan to add velocity scaling to any controller, as that would only result in the desynchronisation of the desired and executed trajectories - if a slower motion is needed, the desired trajectory received by the controller should be slower
